File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]){
        int x, y;
        int count;
        Scanner sc = new Scanner(System.in);
        count = sc.nextInt();
        for (int i = 0; i != count; i++){
            x = sc.nextInt();
            y = sc.nextInt();
            System.out.println(x + y);
        }
    }
}

File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.cc
#include <stdio.h>
int main(){
    int x, y;
    int count;
    scanf("%d\n", &count);
    for(int i = 0 ; i != count; i++){
        scanf("%d\n", &x);
        scanf("%d\n", &y);
        printf("%d\n", x + y);
        }
    }